### PR TITLE
configs for static build pack

### PR DESCRIPTION
**Pull request by recursivefunk**
Including node and static buildpacks

Minimal static config for the heroku static buildpack
- Force https
- Clean URLs (no .html)
- Root is `/public`

Postinstall build

### CODE REVIEW

**Comment by josephabrahams** [comment]
Also closes #54

File: {{ cookiecutter.repo_name }}/gulpfile.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const COMPATIBILITY = ['last 2 versions', 'Firefox ESR', 'not ie <= 10']  // see
 const EXTRAS_GLOB = 'src/**/*.{txt,json,xml,ico,jpeg,jpg,png,gif,svg,ttf,otf,eot,woff,woff2}'
 
 let bundler = browserify({ entry: true, debug: true })
-  .add('src/js/app.js')
+  .add('src/assets/js/app.js')
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Can we change `assets` -> `static` to be consistent with Django/Python projects? @josephabrahams @recursivefunk

File: {{ cookiecutter.repo_name }}/src/templates/404.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+{% raw %}
+<!--
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/src/templates/404.html]: I would like to move this comment into a README or somewhere else. I don't think it belongs in our templates.

**Comment by akrawchyk** [comment]
I'm also interested in merging https://github.com/istrategylabs/mo-static/pull/51 first so we don't have to double-back on updating paths. Not a big deal either way though.

**Comment by akrawchyk** [comment]
Closing in favor of https://github.com/istrategylabs/mo-static/pull/59 which has these updates for the new gulp structure.

**Pull request closed without merging**